src/pi-src/data_collection.cpp

```diff
@@ -14,13 +15,33 @@ Created: 2018-02-10
 #include <stdio.h>
 #include <string.h>
 #include "data_collection.h"
+#include "uid.h"
 #include "spi.h"
 #include "util.h"
 
```

Comment:
At one point there was discussion of having a fifth duration pin. Has this been abandoned?

Reply:
Still possible (see `NUM_PINS` variable), but special handling for it has not been implemented